.cicd/build-scripts.yml

```diff
@@ -48,6 +48,34 @@ steps:
       - thedyrt/skip-checkout#v0.1.1:
           cd: ~
     timeout: 180
+
+
+  - label: ":darwin: macOS 10.15 - Build Pinned"
+    env:
+      REPO: "git@github.com:EOSIO/eos.git"
+      TEMPLATE: "10.15.3_6C_14G_40G"
+      TEMPLATE_TAG: "clean::cicd::git-ssh::nas::brew::buildkite-agent"
+    agents: "queue=mac-anka-large-node-fleet"
+    command:
+      - "git clone git@github.com:EOSIO/eos.git eos && cd eos &&  git checkout -f $BUILDKITE_BRANCH && git submodule update --init --recursive"
+      - "cd eos && ./scripts/eosio_build.sh -P -y"
+    plugins:
+      - EOSIO/anka#v0.6.0:
+          debug: true
+          vm-name: "10.15.3_6C_14G_40G"
+          no-volume: true
+          modify-cpu: 12
+          modify-ram: 24
+          always-pull: true
+          wait-network: true
+          vm-registry-tag: "clean::cicd::git-ssh::nas::brew::buildkite-agent"
+          failover-registries:
+          - "registry_1"
+          - "registry_2"
+          inherit-environment-vars: true
+      - thedyrt/skip-checkout#v0.1.1:
```

Comment:
Should we also have our own fork of this plugin?

Reply:
Good catch! Yes! I'll be more careful to set these properly to our forks.